src/core/utils.js

```diff
@@ -539,6 +553,16 @@ export const validateParam = (param, isXml) => {
       errors.push("Required field is not provided")
       return errors
     }
+    
+    if (maximum) {
+      let err = validateMaximum(value, maximum)
+      if (err) errors.push(err)
+    }
+
+    if (minimum) {
```

Comment:
Will this handle negative minimum values?

Reply:
Sure it will, what make you think it won't?

Reply:
I thought for a second that negative integers would evaluate as false, but I was wrong. The only number that does is `0`.

Reply:
Thanks for adding the test to verify!